pkg/gofr/migration/sql.go

```diff
@@ -21,8 +22,6 @@ const (
     constraint primary_key primary key (version, method)
 );`
 
-	checkSQLGoFrMigrationsTable = `SELECT EXISTS (SELECT 1 FROM information_schema.tables WHERE table_name = 'gofr_migrations');`
-
```

Comment:
this statement will not work when there are multiple databases with the `gofr_migrations` table